### PR TITLE
Feature/add cf cache purge

### PR DESCRIPTION
**Pull request by BradLewis**
Adding a cache purge step to deploy

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -14,3 +14,13 @@ jobs:
       - run: flyctl deploy --remote-only
         env:
           FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+
+  purge:
```
Comment (TheDen) [inline .github/workflows/deploy.yml:18]: Can you ensure this runs only if the `deploy` step passes?

**Review by TheDen — COMMENTED** [review]
Is there a way we can test that the caches are purged?

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (BradLewis) [reply .github/workflows/deploy.yml]: Fixed

**Comment by BradLewis** [comment]
> Is there a way we can test that the caches are purged?
Tested here https://github.com/SongStitch/song-stitch/actions/runs/5097584514/jobs/9164201736 - and all worked as expected

**Review by TheDen — APPROVED** [review]
Nice! LGTM

**Pull request merged**